instructors/instructor-notes.md

```diff
@@ -1,10 +1,19 @@
 ---
-title: FIXME
+title: Trainer Notes
 ---
 
-### Training Themes
+## Schedule
 
-#### Lesson Development as a Learning Process
+The curriculum was piloted in six half-days, 
+with an extended break for lesson trial runs between half-days 4 & 5.
+Below is the schedule for the pilot, showing a break down of episodes
+across each half-day.
```

Comment:
I couldn't find the schedule mentioned below.  Do we still need to add a table or something with the schedule?

That said, do we want to maintain this manually if we rearrange the episodes for each day?